recipe/meta.yaml

```diff
@@ -1,41 +1,59 @@
 {% set name = "jiwer" %}
-{% set version = "2.5.1" %}
+{% set version = "3.0.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jiwer-{{ version }}.tar.gz
-  sha256: 4bfa525c4d5195cd413be4a2b148b7db043dec8c09a5692509504745fc7a0bc6
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: 2438acdc7ca22128fcab4be60db595809d2b5e73785b736de36dc3281a2a6ae8
+  - url: https://github.com/jitsi/{{ name }}/archive/refs/tags/v3.04.tar.gz
+    sha256: d45fccc196468bc64308f517097ba8a49d833581178f5c9ceadbce8518256568
+    folder: upstream_tests
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
 
 requirements:
   host:
-    - python >=3.7,<4.0
+    - python
     - pip
     - poetry-core
   run:
-    - python >=3.7,<4.0
-    - levenshtein ==0.20.2
+    - python
+    - rapidfuzz >=3.0.0,<4.0.0
+    - click >=8.1.3,<9.0.0
 
 test:
+  source_files:
+    - upstream_tests/tests
   imports:
     - jiwer
   commands:
     - pip check
+    - pytest -v upstream_tests/tests
```

Comment:
```suggestion
    - pytest -v upstream_tests/tests -k "not test_speed_"
```

Reply:
These tests are not really time consuming it seems.

Reply:
Benchmarks should not be run. They might be quick today, but they could become very slow tomorrow on the next version. Please remove the benchmarks from the tests.